src/functionalTest/resources/application.yaml

```diff
@@ -14,6 +14,14 @@ azure:
       write-access-key-name: ${PAYMENTS_QUEUE_WRITE_ACCESS_KEY_NAME}
       queue-name: ${PAYMENTS_QUEUE_NAME}
 
+bulk-scan-procesor:
+  api:
+    retries: 10
+    wait-time-in-ms: 1000
+
+async:
```

Comment:
may we can associate this with `bulk-scan-procesor`.

Reply:
associated with bulk-scan-processor